.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,7 @@ blank_issues_enabled: true
 contact_links:
   - name: Discord
     url: https://discord.gg/KDQP7fR467
-    about: 'Join the discord server to contact us directly'
+    about: Join the discord server to contact us directly
+  - name: Email
+    url: mailto:vuetorrent@larsluph.dev?cc=dw.daanwijns@gmail.com
```

Comment:
We probably should only rely on dedicated Vuetorrent email addresses and not personal ones.

Reply:
WDYM by dedicated email address? Do you have one we can use?

Reply:
Would prefer to not have a personal one in there.

Don't really have an alternative ready right now.

Reply:
So I leave only mine?

Reply:
If you're okay with that, yes :)